docs/CommandLine.md

```diff
@@ -0,0 +1,150 @@
+# Command Line
+
+Rokul Runnings has a CLI to execute all commands in the Rokul Runnings class via the command line.
+
+## Setup
+
+### Install
+
+Rokul Runnings CLI can be set up to run either globally or at a project level. In order to use either, they must be installed:
+
+```
+// global
+npm install -g @willowtreeapps/rokul-runnings
+```
+
+```
+// project level
+npm install @willowtreeapps/rokul-runnings
+```
+
+### Defining Config values
+
+Rokul Runnings CLI creates an instance of the Rokul Runnings class based on defaults specified in a config JSON file. The structure of that file looks like:
+
+```json
+{
+  "ip": "", // IP address of the Roku, defaults to "", which will throw an error
+  "username": "rokudev", // username to access Roku Dev portal, defaults to "rokudev"
+  "password": "password", // password to access the Roku Dev portal, defaults to "password",
+  "options": {
+    // Rokul Runnings specific options
+    "pressDelay": 1000, // time to wait in milliseconds between trying keyPresses, defaults to 1000
+    "retryDelay": 1000, // time to wait in milliseconds between retrying calls to the Roku, defaults to 1000
+    "retries": 1 // how many times a call is retried if a non-successful response is returned, defaults to 1
+  },
+  "printOptions": {
+    // options related to how text is printed, optional
+    "true": "", // text decoration to be used for true responses, optional, defaults to green
```

Comment:
`true` and `false` are almost always reserved words, and it's a little jarring to have them as argument names, even under the `printOptions` option. Same thing with `string`, `boolean` and `number` down below. I might append `Color` to the end of all of them, or something similar.

Reply:
Yep, I think I was just using these to keep the differences short and sweet in my mind, but looking back it is pretty jarring. I'll update these.

Reply:
Fixed in 4ae4d34